experiments/billable/scripts/buildVendorBundles.mts

```diff
@@ -40,33 +40,11 @@ const configs = {
         },
       },
     }),
-  reactRSC: (): InlineConfig =>
-    mergeConfig(configs.common(), {
-      resolve: {
-        conditions: ["react-server"],
-      },
-      build: {
-        outDir: DEST_DIR,
-        optimizeDeps: {
-          noDiscovery: false,
-          esbuildOptions: {
-            conditions: ["react-server"],
-          },
-        },
-        lib: {
-          entry: resolve(SRC_DIR, "react-rsc.ts"),
-          name: "react-rsc",
-          formats: ["es"],
-          fileName: "react-rsc",
-        },
-      },
-    }),
```

Comment:
We no longer need that weird rsc bundle of react that I added back [here](https://github.com/redwoodjs/reloaded/pull/36/commits/44632d7973110fe4b72fd4037ff54e1cde31627f#diff-b6d4b9aa1e14ee1b4d3c5faa8433612357289ba0a2afe6d569b7ebd0f75343d6).